running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,201 @@
+# Getting started
+Rent a Ubuntu Server 18.04 or higher from a trusted hosting company such or install for the sake of learning experience Ubuntu Server 18.04 or higher in VirtualBox.
+
+Once you're set we will start by accessing our server. If you're on windows you should get putty(https://www.putty.org/), if you're on macOS or Linux you can use the terminal which is standard available. With putty just follow the screen ( insert your domain/ip-address and click ok ) for macOS and linux users type:
```

Comment:
I think it would be safer to recommend using Windows Subsystem for Linux as a native SSH client or simply installing SSH client via Windows features.

Reply:
Here you have a point indeed! WSL is safer, but putty would be easier for new people. It would be your call what you would like to be better in there.